doc/source/parameters/cfd/post_processing.rst

```diff
@@ -70,7 +74,7 @@ This subsection controls the post-processing other than the forces and torque on
 * ``calculate average velocities``: controls if calculation of time-averaged velocities is enabled.
     * ``initial time``: initial time used for the average velocities calculations.
 
-* ``calculate pressure drop``: controls if calculation of the pressure drop from the inlet boundary to the outlet boundary is enabled. 
+* ``calculate pressure drop``: controls if calculation of the pressure drop from the inlet boundary to the outlet boundary is enabled.
```

Comment:
I would add the formula here for the pressure drop and the "total pressure drop"

Reply:
I would also add the formula for the flow rate.